docs/execution-extras.rst

```diff
@@ -72,31 +72,42 @@ __________
 
 .. automodule:: mpqp.execution.connection.google_connection
 
+.. _cirq-exec:
+
 Execution
 __________
 
 .. automodule:: mpqp.execution.providers.google
 
-.. _con-setup:
+IonQ
+^^^^
 
-Connection setup
-^^^^^^^^^^^^^^^^
+Connection
+__________
+
+.. automodule:: mpqp.execution.connection.ionq_connection
```

Comment:
We didn't add a section for ``Azure`` and ``mpqp.execution.connection.azure_connection", right ?

Reply:
done